dbconn/factory.go

```diff
@@ -111,7 +111,13 @@ func (f factory) Make(cfg blip.ConfigMonitor) (*sql.DB, string, error) {
 
 	// Load and register TLS, if any
 	tlsConfig, err := cfg.TLS.LoadTLS()
-	if tlsConfig != nil && err != nil {
+
+	if err != nil {
+		blip.Debug("Error loading TLS Config for %s", cfg.MonitorId)
```

Comment:
Classic error message mistake: an error that doesn't say what the error is :-)  Either add `err` to the debug, or if the caller debug's the error line, then this Debug is superfluous

Reply:
Yeah thought about it,  i will be printing the error twice, so didn't print it here , caller will be logging the returned error anyway.

Reply:
Cleaned up by removing the whole debug, also I was looking for patterns to say logger.Error or logger.Warn but I only see them available in few file(main.go or blip.go) .Is that intentional? Most other files only have blip.Debug available.